app/components/on-this-page-links.js

```diff
@@ -0,0 +1,17 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+
+export default class OnThisPageLinksComponent extends Component {
+  @action
+  scrollToFragment(fragName, e) {
+    // chrome does not work correctly with fragment scrolling, custom override needed
```

Comment:
I know @ValenberghsSven did some work to fix this recently for the other subcase views. Is this double, or something else?

Reply:
Aha, I see this functionality was moved from the subcase view to this component. So am I correct to assume it includes the earlier fix?